ci_jobs_trigger/libs/openshift_ci/zstream_trigger/zstream_trigger.py

```diff
@@ -239,7 +238,15 @@ def process_and_trigger_jobs(logger: logging.Logger, version: str | None = None)
                 if _rosa_env
                 else get_accepted_cluster_versions()
             )
-            _latest_version = _all_versions.get(_version_channel)[_wanted_version][0]
+
+            if not (wanted_version_list := _all_versions.get(_version_channel).get(_wanted_version)):
```

Comment:
This `_all_versions.get(_version_channel)` can fail ,you need to set default 
`_all_versions.get(_version_channel, {})`

Reply:
Updated